scripts/download-pokemon-images.js

```
@@ -0,0 +1,290 @@
+/**
+ * Download Pokemon TCG card images
+ * This script downloads card images for the most recent sets to provide initial caching
+ *
+ * It uses a caching strategy to avoid downloading images that are already cached,
+ * which is important for frequent builds.
+ */
+
+const fs = require('fs');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)